ttnn/cpp/ttnn/operations/transformer/sdpa_decode/device/kernels/compute/sdpa_flash_decode.cpp

```diff
@@ -506,6 +506,11 @@ void MAIN {
         cur_pos = index_addr_ptr[4+cur_batch];
         cb_release_tile(cb_index_id);
     }
+
+    if (cur_pos == (uint32_t) -1) {
```

Comment:
This looks kind of weird. Maybe make `4294967295` some constexpr arg (something like `UINT32_MAX`)?

Reply:
Can do, thanks Brian